doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -7,9 +7,7 @@
 QoS support in Ceph is implemented using a queuing scheduler based on `the
 dmClock algorithm`_. See :ref:`dmclock-qos` section for more details.
 
-.. note:: The *mclock_scheduler* is supported for BlueStore OSDs. For Filestore
-          OSDs the *osd_op_queue* is set to *wpq* and is enforced even if you
-          attempt to change it.
+.. note:: The *mclock_scheduler* is supported for BlueStore OSDs.
```

Comment:
Does leaving the residual note make any sense?

Reply:
Agree. removing